src/Stryker.CLI/Stryker.CLI/runtimeconfig.template.json

```diff
@@ -1,3 +1,3 @@
 {
-  "rollForwardOnNoCandidateFx": 2
+  //"rollForwardOnNoCandidateFx": 2
```

Comment:
Accidental commit?

Reply:
yes, sorry about that. It got embarked with the merge, I guess.
By the way, the problem is that VS 17.7 does add RollForward on its own, so it conflicts with this. So far, I failed to find anything to either control this setting or ensure a correct result

Reply:
Yeah btw, so this is not only me, had to do a similar hack locally. `rollForward` also works...